src/nidigital/metadata/config.py

```diff
@@ -44,6 +44,16 @@
             'prefix': '',
             'python_name': 'pins',
         },
+        # These need to be added here since merging 'repeated_capabilities' with something
+        # here and something in config_addons.py doesn't work
+        {
+            'prefix': 'conditionalJumpTrigger',
+            'python_name': 'conditional_jump_trigger',
+        },
+        {
+            'prefix': 'patternOpcodeEvent',
+            'python_name': 'pattern_opcode_event',
```

Comment:
`python_name`s should be plural. Also update Changelog.

Reply:
Both updated.